Revert commit 8f182270dfec mm/swap.c: flush lru pvecs on compound page arrival to fix OOM error.

Signed-off-by: Steven Rostedt <rostedt <at> goodmis.org>
---
diff --git a/mm/swap.c b/mm/swap.c
index b523f0a4cbfb..ab3b9c2dd783 100644
--- a/mm/swap.c
+++ b/mm/swap.c
 <at>  <at>  -631,7 +631,7  <at>  <at>  static void __lru_cache_add(struct page *page)
    struct pagevec *pvec = &get_cpu_var(lru_add_pvec);

    page_cache_get(page);
-   if (!pagevec_space(pvec) || PageCompound(page))
+   if (!pagevec_add(pvec, page) || PageCompound(page))
        __pagevec_lru_add(pvec);
    put_cpu_var(lru_add_pvec);
 }
